src/model/exec-with-error-check.ts

```diff
@@ -12,18 +13,35 @@ export async function execWithErrorCheck(
     return result.exitCode;
   }
 
-  // Check for errors in the Build Results section
-  const match = result.stdout.match(/^#\s*Build results\s*#(.*)^Size:/ms);
+  // Check for errors in the Unity Build Results section
+  const unityMatch = result.stdout.match(/^#\s*Build results\s*#(.*)^Size:/ms);
 
-  if (match) {
-    const buildResults = match[1];
+  if (unityMatch) {
+    const buildResults = unityMatch[1];
     const errorMatch = buildResults.match(/^Errors:\s*(\d+)$/m);
     if (errorMatch && Number.parseInt(errorMatch[1], 10) !== 0) {
       throw new Error(`There was an error building the project. Please read the logs for details.`);
     }
-  } else {
-    throw new Error(`There was an error building the project. Please read the logs for details.`);
+    await summary.addHeading('Build Results').addQuote(unityMatch[0]).write();
+
+    return result.exitCode;
+  }
+
+  // Check for presence of game-ci message(s)
+  const gameciMatch = result.stdout.match(/^GAME_CI_BUILD_SUCCESS$/ms);
```

Comment:
I really like the idea of having a very simple message to check for. We should also document this to make sure, people using this for non-build stuff are aware.

Additionally, I'm wondering if there could be use cases, where the rather explicit `GAME_CI_BUILD_SUCCESS` should have a higher priority than `Build results` (You want to have the job succeed even if the Build results contain an error).